audio/audacity/Portfile

```diff
@@ -142,26 +138,33 @@ proc add_patchfile_ifexists {args} {
     }
 }
 
-# openssl is a dependency of curl in cmake and of python3.7
-license_noconflict  cmake python37
+# openssl is a dependency of curl in cmake and (indirectly) of git, both build dependencies
+license_noconflict  openssl
```

Comment:
isn't openssl v3 used for these build dependencies? Of so, there shouldn't be a conflict anymore and the line is redundant?